packages/api/portfolioDrafts/funding/createFundingStep.ts

```diff
@@ -100,112 +67,18 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
   }
   return new ApiSuccessResponse<FundingStep>(fundingStep, SuccessStatusCode.CREATED);
 }
+const handler = middy(baseHandler);
+handler
+  .use(xssSanitizer())
+  .use(jsonBodyParser())
+  .use(
+    validator({
+      inputSchema: wrapSchema(schema.FundingStep),
+      ajvOptions: { strict: false },
+    })
+  )
+  .use(errorHandlingMiddleware())
+  .use(JSONErrorHandlerMiddleware())
+  .use(cors({ headers: "*", methods: "*" }));
```

Comment:
Just want to point out that this relates to the comments at https://github.com/dod-ccpo/atat-web-api/pull/381#discussion_r742990896

Reply:
Thanks for linking to that.